scripts/build/termux_step_install_license.sh

```diff
@@ -1,87 +1,85 @@
+# shellcheck shell=bash
 termux_step_install_license() {
-	[ "$TERMUX_PKG_METAPACKAGE" = "true" ] && return
+	[[ "$TERMUX_PKG_METAPACKAGE" == 'true' ]] && return
```

Comment:
Use `"true"` like everywhere else.

Reply:
They are equivalent.
Since `true` is a static string I used single quotes,
which are not subject to interpolation, unlike double quoted strings.

Reply:
I meant in all other build scripts we always use double quotes and also a single `[`, so best stay consistent. Only python scripts use single quotes.

Reply:
We also make extensive use of Bash specific features such as arrays.
I don't think we should cling to such POSIX-isms if our build scripts are written for Bash.

Reply:
It's not a hill I'm gonna choose to die on,
but `[[` bash test `]]` has much better defined semantics, condition chaining, additional test operators and regex support.

See the Wooledge Bash Wiki article on conditionals from about this paragraph.
https://mywiki.wooledge.org/BashGuide/TestsAndConditionals#Conditional_Blocks_.28if.2C_test_and_.5B.5B.29:~:text=pitfall%20when%20%5B%20is%20used

<h1><em></em></h1> <!-- thin separator -->

The quoting I can go either way on, I just prefer single quoting static strings (and fwiw `shellharden` agrees with that stance)

Reply:
Oh, its fine using `[[`, I use the same, unless writing posix compatible scripts. that's why I initially just mentioned double quotes. There is no instance of `'true'` check in any bash shell script in the entire repo.